views/head.jade

```diff
@@ -7,15 +7,15 @@ link(rel='icon', type='image/png', href='/images/icon.png')
 
 // Stylesheets too include
 link(rel='stylesheet', href="https://fonts.googleapis.com/css?family=Open+Sans|Russo+One|Ubuntu+Mono")
-link(rel='stylesheet', href="//cdnjs.cloudflare.com/ajax/libs/normalize/2.1.3/normalize.min.css")
-link(rel='stylesheet', href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.0-rc2/css/bootstrap#{min}.css")
+link(rel='stylesheet', href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.2/normalize.min.css")
```

Comment:
I didn't know what `#{min}` was doing and I made `https:` explicit because... there was a better line of reasoning when I did it, but there ain't much of a reason.